pkgs/tools/wayland/wtype/default.nix

```diff
@@ -10,18 +11,18 @@
 , wayland
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "wtype";
-  version = "2020-09-14";
+  version = "0.3";
 
   src = fetchFromGitHub {
     owner = "atx";
     repo = "wtype";
-    rev = "74071228dea4047157ae82960a2541ecc431e4a1";
-    sha256 = "1ncspxpnbwv1vkfmxs58q7aykjb6skaa1pg5sw5h798pss5j80rd";
+    rev = "v${version}";
+    hash = "sha256-8q2PxI3ItR4tsP/KOiSlqnuK4ZSe7OdekheolEFVmys=";
   };
 
-  nativeBuildInputs = [ meson ninja pkg-config wayland ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland git ];
```

Comment:
Why is it requiring git? We do not have any git directory in the source.

Reply:
It looks like the wtype meson build uses `git`: <https://github.com/atx/wtype/blob/master/meson.build#L11>.

Reply:
I didn't realize it was an optional dependency. I vaguely recall nix-build failed without it, but after removing nix-build is still happy and the app works (for my limited test of just a couple of keystrokes).

I've updated the merge request accordingly.